## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 455690`
Commit: `3af77aaf905eeb25b0407c78f2cbc17fc3cc251e`

---
### `x86_64-linux`
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>fcitx5-openbangla-keyboard</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 26 packages built:</summary>
  <ul>
    <li>fcitx5</li>
    <li>fcitx5-anthy</li>
    <li>fcitx5-bamboo</li>
    <li>fcitx5-chewing</li>
    <li>fcitx5-gtk</li>
    <li>fcitx5-gtk.dev</li>
    <li>fcitx5-hangul</li>
    <li>fcitx5-lua</li>
    <li>fcitx5-m17n</li>
    <li>fcitx5-mcbopomofo</li>
    <li>fcitx5-mozc</li>
    <li>fcitx5-mozc-ut</li>
    <li>fcitx5-rime</li>
    <li>fcitx5-skk</li>
    <li>fcitx5-table-extra</li>
    <li>fcitx5-table-other</li>
    <li>kdePackages.fcitx5-chinese-addons (qt6Packages.fcitx5-chinese-addons)</li>
    <li>kdePackages.fcitx5-configtool (qt6Packages.fcitx5-configtool)</li>
    <li>kdePackages.fcitx5-qt (qt6Packages.fcitx5-qt)</li>
    <li>kdePackages.fcitx5-skk-qt (qt6Packages.fcitx5-skk-qt)</li>
    <li>kdePackages.fcitx5-unikey (qt6Packages.fcitx5-unikey)</li>
    <li>kdePackages.fcitx5-with-addons (qt6Packages.fcitx5-with-addons)</li>
    <li>libime</li>
    <li>libsForQt5.fcitx5-qt (plasma5Packages.fcitx5-qt)</li>
    <li>mlterm</li>
    <li>mlterm-wayland</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>fcitx5-openbangla-keyboard</summary>
<pre>patching file src/shared/FileSystem.cpp
Executing cargoSetupPostPatchHook
Validating consistency between /build/source/src/engine/riti/Cargo.lock and /build/cargo-deps-vendor/Cargo.lock
Finished cargoSetupPostPatchHook
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;updateAutotoolsGnuConfigScriptsPhase&quot; }
Running phase: configurePhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;configurePhase&quot; }
cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/83zi6rlzz4bz3amc4v2801ffmqlqbhkw-openbangla-keyboard-unstable-2023-07-21/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/83zi6rlzz4bz3amc4v2801ffmqlqbhkw-openbangla-keyboard-unstable-2023-07-21/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/83zi6rlzz4bz3amc4v2801ffmqlqbhkw-openbangla-keyboard-unstable-2023-07-21/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/83zi6rlzz4bz3amc4v2801ffmqlqbhkw-openbangla-keyboard-unstable-2023-07-21/share/doc/OpenBangla-Keyboard -DCMAKE_INSTALL_INFODIR=/nix/store/83zi6rlzz4bz3amc4v2801ffmqlqbhkw-openbangla-keyboard-unstable-2023-07-21/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/83zi6rlzz4bz3amc4v2801ffmqlqbhkw-openbangla-keyboard-unstable-2023-07-21/share/man -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/83zi6rlzz4bz3amc4v2801ffmqlqbhkw-openbangla-keyboard-unstable-2023-07-21/include -DCMAKE_INSTALL_SBINDIR=/nix/store/83zi6rlzz4bz3amc4v2801ffmqlqbhkw-openbangla-keyboard-unstable-2023-07-21/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/83zi6rlzz4bz3amc4v2801ffmqlqbhkw-openbangla-keyboard-unstable-2023-07-21/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/83zi6rlzz4bz3amc4v2801ffmqlqbhkw-openbangla-keyboard-unstable-2023-07-21/lib -DCMAKE_POLICY_DEFAULT_CMP0025=NEW -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_STRIP=/nix/store/x8mydcgbry214s802nzvy7fdljx404ym-gcc-wrapper-14.3.0/bin/strip -DCMAKE_RANLIB=/nix/store/x8mydcgbry214s802nzvy7fdljx404ym-gcc-wrapper-14.3.0/bin/ranlib -DCMAKE_AR=/nix/store/x8mydcgbry214s802nzvy7fdljx404ym-gcc-wrapper-14.3.0/bin/ar -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DCMAKE_INSTALL_PREFIX=/nix/store/83zi6rlzz4bz3amc4v2801ffmqlqbhkw-openbangla-keyboard-unstable-2023-07-21 -DENABLE_FCITX=YES
CMake Error at CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake &lt; 3.5 has been removed from CMake.

  Update the VERSION argument &lt;min&gt; value.  Or, use the &lt;min&gt;...&lt;max&gt; syntax
  to tell CMake that the project requires at least &lt;min&gt; but has been updated
  to work with policies introduced by &lt;max&gt; or earlier.

  Or, add -DCMAKE_POLICY_VERSION_MINIMUM=3.5 to try configuring anyway.


-- Configuring incomplete, errors occurred!</pre>
</details>
</details>

--- a/pkgs/tools/inputmethods/fcitx5/fcitx5-configtool.nix
+++ b/pkgs/tools/inputmethods/fcitx5/fcitx5-configtool.nix
@@ -31,13 +31,13 @@
 
 stdenv.mkDerivation rec {
   pname = "fcitx5-configtool";
-  version = "5.1.10";
+  version = "5.1.11";
 
   src = fetchFromGitHub {
     owner = "fcitx";
     repo = pname;
     rev = version;
-    hash = "sha256-Py2UDBQRqvT7kwZeQIXKrIjGAbOjjxEyEfO5tdtizW4=";
+    hash = "sha256-SEQelUMigcqs0C+jY+A/dfncEogzU1e5tjP+wK+MylM=";
   };
 
   cmakeFlags = [

--- a/pkgs/tools/inputmethods/fcitx5/fcitx5-m17n.nix
+++ b/pkgs/tools/inputmethods/fcitx5/fcitx5-m17n.nix
@@ -14,13 +14,13 @@
 
 stdenv.mkDerivation rec {
   pname = "fcitx5-m17n";
-  version = "5.1.4";
+  version = "5.1.5";
 
   src = fetchFromGitHub {
     owner = "fcitx";
     repo = pname;
     rev = version;
-    hash = "sha256-TJMJGjO9V6EOzxt6Z7rwOfIQWK38XolDhUKbjbNUGhA=";
+    hash = "sha256-gvR//H+Kf/pYv6Avr0No8PZO7hAnOHj16v6n+tDXgkU=";
   };
 
   nativeBuildInputs = [

--- a/pkgs/tools/inputmethods/fcitx5/fcitx5-skk.nix
+++ b/pkgs/tools/inputmethods/fcitx5/fcitx5-skk.nix
@@ -16,13 +16,13 @@
 
 stdenv.mkDerivation rec {
   pname = "fcitx5-skk";
-  version = "5.1.7";
+  version = "5.1.8";
 
   src = fetchFromGitHub {
     owner = "fcitx";
     repo = pname;
     rev = version;
-    hash = "sha256-WMkcZSocanhWMn9kiWyB07jEW4x84G07kAYvn5heenc=";
+    hash = "sha256-1omxT31hKe7gQ5BARJ+0tIp4RT5eM+Tjufd6s/PxBoY=";
   };
 
   nativeBuildInputs = [

